File: Domain/Models/CategoryService.cs
namespace Domain.Models
{
    public class CategoryService
    {
        public CategoryService()
        {
            Id = Guid.NewGuid();
            Services = new List<Service>();

        }

        public Guid Id { get; set; }
        public string Name { get; set; }
        public string UrlCategoryserviceIcon { get; set; }
        public bool IsDeleted { get; set; } = false;
        public bool IsAccepted { get; set; } = false;
        public Guid SectorId { get; set; }
        public Sector Sector { get; set; }
        public ICollection<Service> Services { get; set; }
    }
}

File: ScheduledServicesCenterUI/src/models/category.service.ts
import { Guid } from 'guid-typescript';
import { Sector } from './sector';
import { Service } from './service';


export class CategoryService {
  Id: Guid | undefined;
  Name: string | undefined;
  UrlCategoryserviceIcon: string | undefined;
  IsDelete: boolean | undefined;
  IsAccepted: boolean | undefined;
  SectorId: Guid | undefined;
  Sector: Sector | undefined;
  Services: Service[] | undefined;
  /**
   *
   */
}
